README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/adilfulara/handsome.svg?branch=test-setup)](https://travis-ci.org/adilfulara/handsome)
+[![Coverage Status](https://coveralls.io/repos/github/adilfulara/handsome/badge.svg?branch=test-setup)](https://coveralls.io/github/adilfulara/handsome?branch=test-setup)
```

Comment:
Would likely want to change the username on these to `davefp` before merging for practical reasons.